wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/PerpetualCommand.java

```diff
@@ -50,4 +50,9 @@ public void end(boolean interrupted) {
   public boolean runsWhenDisabled() {
     return m_command.runsWhenDisabled();
   }
+
+  @Override
+  public PerpetualCommand perpetually() {
+    return this;
```

Comment:
Is this supposed to do something more than just return this?

Reply:
Wrapping one PerpetualCommand with another is redundant- the behavior is identical. For that reason, I dropped the outer layer.

If a team finds a specific use-case that they need two nested PerpetualCommands, they can use the constructor.